src/app/api/delegate/payment/route.ts

```diff
@@ -82,20 +83,20 @@ export async function POST(req: Request) {
 
       let transaction;
 
+      const talentLayerClient = initializeTalentLayerClient();
```

Comment:
a few lines above `const walletClient = await getDelegationSigner();` is only being used for an if statement to throw an error
We can get rid of it. Instead replace it with a check on `if(const talentLayerClient )`